### DISCUSSION TITLE
Non-NextJS Compatibility

### DISCUSSION THREAD

**Discussion by ghost** [fuma-nama/fumadocs#1030] [Q&A] [answered]
Does anyone know if it's possible to use this with Remix or TanStack Start? 1 thing that's NextJS specific is Link component, is there a way to override this via configuration? Thanks.

**Comment by fuma-nama** [comment, marked as answer]
this isn’t possible, you can use some primitive from fumadocs core, but it’s not designed for framework agnostic use

**Reply by ghost** [reply]
Oh ok thx!
